kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/HbaseSinkTask.scala

```diff
@@ -60,11 +64,19 @@ class HbaseSinkTask extends SinkTask with StrictLogging {
       context.timeout(sinkConfig.getInt(HBaseConfigConstants.ERROR_RETRY_INTERVAL).toLong)
     }
 
+    val hbaseConf = ConfigurationBuilder.buildHBaseConfig(hbaseSettings)
```

Comment:
**hbaseConf** holds the hbase-client connection specific configuration parameters. I believe it's cleaner to avoid mixing user configuration settings with client configuration. Still I can do this change if you believe the opposite.